spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraSQLContext.scala

```diff
@@ -37,6 +41,152 @@ class CassandraSQLContext(sc: SparkContext) extends SQLContext(sc) {
   @transient
   val conf = sc.getConf
 
+  @transient
+  private val clusterReadConf: Map[String, SparkConf] = Map()
+  @transient
+  private val keyspaceReadConf: Map[Seq[String], SparkConf] = Map()
```

Comment:
It's more straight forward to use different map instead of using magic characters.